worlds/witness/player_items.py

```diff
@@ -100,6 +100,42 @@ def __init__(self, world: "WitnessWorld", player_logic: WitnessPlayerLogic,
             self.item_data[location_name] = ItemData(None, ItemDefinition(0, ItemCategory.EVENT),
                                                      ItemClassification.progression, False)
 
+        # Determine which items should be progression + useful, if they exist in some capacity.
+        # Note: Some of these may need to be updated for the "independent symbols" PR.
+        self._proguseful_items = {
+            "Dots", "Stars", "Shapers", "Black/White Squares",
+            "Caves Shortcuts", "Caves Mountain Shortcut (Door)", "Caves Swamp Shortcut (Door)",
+            "Boat",
+            "Town Obelisk Key",  # Most checks
```

Comment:
Does this account for Obelisk Sides, where well, it's just not the most checks

Reply:
Good point. I'll have to change that.